files/packit.wsgi

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+
+import logging
+from packit.service import web_hook
+
+logging.basicConfig(level="DEBUG")
```

Comment:
please use set_logging from utils

Reply:
I have removed it. It does not make sense for now.